CONTRIBUTING.md

```diff
@@ -0,0 +1,170 @@
+# Contributing to PivorakWebApp
+:+1::tada: Greetings, contributor! From the #pivorak Ruby usergroup we want to say thank you for joining us on this path to community development. We appreaciate your effort, every change counts!
+ :tada::+1:
+
+The following is a set of guidelines for contributing to PivorakWebApp. We've tried our best to ease this process for you, so please read it carefully.
+
+#### Table Of Contents
+- [Code of conduct](https://pivorak.com/code-of-conduct)
+- [How to setup project](#how-to-setup-project)
+    - [Option 1: Without docker](#option-1-without-docker)
+    - [Option 2: With docker](#option-2-with-docker)
+- [How to report a bug](#how-to-report-a-bug)
```

Comment:
Link to How to contribute is missing. 
Move how to report a bug at the end of the table of content to inline with actual paragraphs order